tests/puppeteer/campaigns/sanity/productsBO/03_CRUDStandardProductWithCombinationsInBO.js

```diff
@@ -0,0 +1,84 @@
+// Using chai
+const {expect} = require('chai');
+
+// importing pages
+const LoginPage = require('../../../pages/BO/login');
+const DashboardPage = require('../../../pages/BO/dashboard');
+const BOBasePage = require('../../../pages/BO/BObasePage');
+const ProductPage = require('../../../pages/BO/product');
+const AddProductPage = require('../../../pages/BO/addProduct');
+const FOProductPage = require('../../../pages/FO/product');
+const ProductFaker = require('../../data/faker/product');
+
+
+let page;
+let loginPage;
+let dashboardPage;
+let boBasePage;
+let productPage;
+let addProductPage;
+let foProductPage;
+let productWithCombinations;
+let editedProductWithCombinations;
+
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  loginPage = await (new LoginPage(page));
+  dashboardPage = await (new DashboardPage(page));
+  boBasePage = await (new BOBasePage(page));
+  productPage = await (new ProductPage(page));
+  addProductPage = await (new AddProductPage(page));
+  foProductPage = await (new FOProductPage(page));
+  productWithCombinations = await (new ProductFaker('Standard product', false, '1', false, true));
```

Comment:
It's not very easy to know what are all these arguments:
```false, '1', false, true```
means we need to go to the function declaration to know what is what. Maybe you could create an object to pass instead of a few parameters ? Also it's dangerous if we want to change these in the future.
What do you think ?
```suggestion
  let productToCreate = {
  name : 'Standard product',
  quantity: false,
  wantedQuantity: '1',
  price: false,
  productHasCombinations : true
  }
  productWithCombinations = await (new ProductFaker(productToCreate));
```

Reply:
I'm thinking more this :
for Simple product
```js 
  let productToCreate = {
    type : 'Standard product',
    productHasCombinations : false,
  };
  productData = await (new ProductFaker(productToCreate));
  editedProductData = await (new ProductFaker(productToCreate));
```
And for product with combinations : 
```js 
  let productToCreate = {
    type : 'Standard product',
    productHasCombinations : true,
  };
  productData = await (new ProductFaker(productToCreate));
  editedProductData = await (new ProductFaker(productToCreate));
```
more simple to add only attributes needed in `productToCreate`

Reply:
done thanks